.cirrus.yml

```
@@ -62,6 +62,7 @@ task:
   << : *BASE_TEMPLATE
   container:
     image: ubuntu:bionic # For python 3.6, oldest supported version according to doc/dependencies.md
+    cpu: 1 # Cut bill in half for linting
```

No need for that comment. Can remove or just say 1 is sufficient

```suggestion
    cpu: 1
```

```suggestion
    cpu: 1  # No need for more CPUs
```

---

btw, based on this plot https://cirrus-ci.com/task/4624095184683008

memory can also be set to 1